static/js/main.js

```diff
@@ -0,0 +1,49 @@
+const toggleClass =  (el, cls) => {
+    if (Array.isArray(cls)) {
+        cls.map((cl) => {
+            el.classList.toggle(cl);
+        });
+    } else {
+        el.classList.toggle(cls);
+    }
+};
+
+function DarkModeIndex() {
+    var footer = document.getElementById('footer')
+    footer.classList.toggle("footer-dark")
+
+    toggleClass(document.getElementById('navbar'), ['bg-dark', 'navbar-dark']);
+    
+    toggleClass(document.getElementById('body'), ['body-dark']);
+    toggleClass(document.getElementById('darkmode'), ['body-dark']);
+    toggleClass(document.getElementById('title'), ['title-dark']);
+    toggleClass(document.getElementById('parse'), ['btn-primary']);
+    toggleClass(document.getElementById('container'), ['container-dark']);
+    toggleClass(document.getElementById('subtitle'), ['subtitle-dark']);
+  }
+
+
+function DarkModeShow() {
+    var footer = document.getElementById('footer')
+    footer.classList.toggle("footer-dark")
+
+    toggleClass(document.getElementById('navbar'), ['bg-dark', 'navbar-dark']);
+    toggleClass(document.getElementById('info1'), ['subtitle-dark']);
+    toggleClass(document.getElementById('info2'), ['subtitle-dark']);
+    toggleClass(document.getElementById('info3'), ['subtitle-dark']);
+    toggleClass(document.getElementById('info4'), ['subtitle-dark']);
+    toggleClass(document.getElementById('error'), ['error-dark']);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/zlataovce/plshelp/pullRequest?prid=7177914)
```suggestion
    toggleClass(document.getElementById("error"), ['error-dark']);
```